src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Mapping/ProductMappingProfile.cs

```diff
@@ -28,59 +24,9 @@ public ProductMappingProfile()
             CreateMap<LoadCategoryQuery, SearchCategoryQuery>();
 
             CreateMap<SearchProductAssociationsQuery, ProductAssociationSearchCriteria>();
-            CreateMap<SearchDocument, ExpProduct>().ConvertUsing((src, dest, context) =>
-            {
-                if (!context.Items.TryGetValue("store", out var storeObj))
-                {
-                    throw new OperationCanceledException("store must be set");
-                }
-                var store = storeObj as StoreModule.Core.Model.Store;
-                context.Items.TryGetValue("language", out var language);
-                language ??= store.DefaultLanguage;
-
-                var genericModelBinder = new GenericModelBinder<ExpProduct>();
-                var result = genericModelBinder.BindModel(src) as ExpProduct;
-                if (result != null)
-                {
-                    result.AllPrices = context.Mapper.Map<IEnumerable<ProductPrice>>(result.IndexedPrices).ToList();
```

Comment:
Is this code executed somewhere now? Or is it useless?

Reply:
> Is this code executed somewhere now?
In the second pipeline.
> Or is it useless?
Code duplication.